test/ReactWrapper-spec.jsx

```diff
@@ -385,6 +394,8 @@ describeWithDOM('mount', () => {
     });
 
     it('should not find components with invalid attributes', () => {
+      // React 15.2 warns when setting a non valid prop to an DOM element
+      sandbox.stub(console, 'error');
```

Comment:
If I haven't already, I'll soon be making console warnings and errors fail tests - so if a test would be issuing a warning in a version of React, we shouldn't be running that test in that version of React (since React covers it for us by issuing the warning).

Can we restrict this test to only React < 15, and remove the stub?